app/models/cloud_object_store_container/operations.rb

```diff
@@ -8,4 +8,12 @@ def delete_cloud_object_store_container
   def raw_delete
     raise NotImplementedError, _("must be implemented in subclass")
   end
+
+  def cloud_object_store_container_empty
+    raw_cloud_object_store_container_empty
+  end
+
+  def raw_cloud_object_store_container_empty
+    raise NotImplementedError, _("must be implemented in subclass")
```

Comment:
It looks like the AWS API call is `clear`.  I think that's more descriptive than `empty`.

My first impression with `cloud_object_store_container_emtpy` was that it was missing a question mark and it should be predicate method.

I think `cloud_object_store_container_clear` sounds like it's going to clear the container.

Or, do what @agrare said, and reverse the naming by putting the verb before the object: `empty_cloud_object_store_container`.  In either case, I lean closer to sticking with `clear` instead of `empty`.  But, I could live with either if the verb comes first.